api/swagger-spec/swagger.yml

```diff
@@ -52,6 +52,26 @@ paths:
           description: unexpected error
           schema:
             $ref: "#/definitions/error"
+  /v3/doctor/{train}/{release}/{uuid}:
```

Comment:
@slack do we want to concern ourselves namespacing deis doctor submissions with version info (`train` and `release` here)? we can probably get away with just `/v3/doctor/{uuid}` (where `uuid` is the id of the generated deis doctor debug data)